chia/full_node/full_node.py

```diff
@@ -1306,16 +1303,10 @@ async def add_block_batch(
                     if block_rec.sub_epoch_summary_included.new_difficulty is not None:
                         vs.current_difficulty = block_rec.sub_epoch_summary_included.new_difficulty
 
-            if fork_info is None:
-                continue
             # the below section updates the fork_info object, if
             # there is one.
-
-            # TODO: it seems unnecessary to request overlapping block ranges
-            # when syncing
```

Comment:
maybe you know the answer to this. is it because we can only reorg at the start of a sub-epoch or epoch? So we always overshoot?